hack/test

```diff
@@ -44,7 +44,24 @@ function build_pass {
 	IMAGE=$OPERATOR_IMAGE hack/build/operator/build
 }
 
+function rbac_cleanup {
+	kubectl delete clusterrolebinding etcd-operator
+	kubectl delete clusterrole etcd-operator
+}
+
 function e2e_pass {
+	# Enable rbac if present
+	if kubectl --kubeconfig $KUBECONFIG get clusterrole 1> /dev/null ; then
+		trap rbac_cleanup EXIT
+		# Create role and binding
+		if "hack/ci/create_rbac_role"; then
```

Comment:
Can we have the other file to be a util, e.g. rbac_util.sh? Then `source xxx`.

It should contain two funcs, setup_rbac() and cleanup_rbac().